connections/debug/debug_connection.py

```diff
@@ -126,6 +127,8 @@ def config_mock_set_values(self) -> bytearray:
     def event_mock_set_values(self) -> bytearray:
         return radio_packets.event(self._current_millis(), 0x00)
 
+    def state_mock_set_values(self) -> bytearray:
+        return radio_packets.state(self._current_millis(), 0x05)
```

Comment:
Is there an easy way to grab the name of the state, rather than the magic value?

Reply:
To go by some enum entry we'd have to either create a local or global reversed dictionary, because the dictionary is number -> DataEntryValue. So this is what you'd prefer? I personally think it's overkill, since it is also the debug connection testing some random state value.
```suggestion
        reversed_dict = {v: k for k, v in STATE_IDS.items()}
        return radio_packets.state(self._current_millis(), reversed_dict[DataEntryValues.STATE_MACH_LOCK])
```